context/app/static/js/pages/Collection/Collection.jsx

```diff
@@ -50,7 +49,6 @@ function Collection({ collection: collectionData }) {
               </OutboundIconLink>
             )}
           </Summary>
-          {'contacts' in collectionData && <ContributorsTable contributors={contacts} title="Contacts" />}
           {'datasets' in collectionData && <CollectionDatasetsTable datasets={datasets} />}
           {'creators' in collectionData && <ContributorsTable contributors={creators} title="Contributors" />}
```

Comment:
I may be missing some context - do we want to display both contacts and contributors for collections? I was under the impression that contacts are a subset of the contributors list, and that the contributors list now indicates the contacts; are there cases where contacts are not contributors?

Reply:
Isn't this removing contacts?

Reply:
Yup, I misread this for some reason, good call 😆 Friday is taking its toll